localisation/00_zuwc_united_localisation.csv

```diff
@@ -2988,10 +2988,10 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
 legion_is_destroyed_tooltip;The Burning Legion has NOT fallen;;;;;;;;;;;;;x
 RULE_OPTION_INFERTILE;Infertile;;;;;;;;;;;;;x
 RULE_GENERATE_FAMILIES_INFERTILE_DESC;Most infertile landed historical characters with no recorded family will have a spouse and up to four children generated for them upon starting a new game. This reduces the risk of an early game over.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+convert_to_local_culture;Convert to [Root.Capital.Culture.GetName];;;;;;;;;;;;;x
+EVTDESC_WOL_2080_DW;I can hardly believe it, but I have uncovered indisputable proof that [FromFromFrom.GetTitledName] does not follow what they preach and instead secretly follows the philosophies and teachings of  [FromFromFrom.Religion.GetRandomEvilGodName]! [FromFromFrom.GetSubjectPronounCap] is a member of an organization called [FromFromFrom.Society.GetName]. How should I choose to utilize this explosive information?;;;;;;;;;;;;;x
```

Comment:
@DukeJeans , wait, `GetRandomEvilGodName` is still here. So, https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/900 isn't fixed.

Reply:
👌